frontend/www/js/omegaup/components/arena/ContestListv2.vue

```diff
@@ -366,6 +371,15 @@ export default class ArenaContestList extends Vue {
     }
   }
 
+  numberOfPages(tab: ContestTab) {
```

Comment:
This function will return a number, so you need to define it:
```suggestion
  numberOfPages(tab: ContestTab): number {
```